system-variables.md

```diff
@@ -1566,73 +1559,63 @@ SET tidb_slow_log_threshold = 200;
+- Scope: GLOBAL
+- Default value: `OFF`
+- This variable controls how TiDB behaves when the waiting time of SQL optimization reaches the timeout to synchronously load complete column statistics. The default value `OFF` means that SQL execution fails after the timeout. If you set this variable to `ON`, the SQL optimization gets back to using pseudo statistics after the timeout.
 
-### `tidb_stats_load_pseudo_timeout` <span class="version-mark">New in v5.4.0</span>
+### tidb_stats_load_sync_wait <span class="version-mark">New in v5.4.0</span>
 
-> **WARNING:**
+> **Warning:**
 >
 > Currently, synchronously loading statistics is an experimental feature. It is not recommended that you use it in production environments.
 
-- Scope: GLOBAL
-- Default value: `OFF`
-- This variable controls how TiDB behaves when the waiting time of SQL optimization reaches the timeout to synchronously load complete column statistics. The default value `OFF` means that SQL execution fails after the timeout. If you set this variable to `ON`, the SQL optimization gets back to using pseudo statistics after the timeout.
+- Scope: SESSION | GLOBAL
+- Default value: `0`
+- Range: `[0, 2147483647]`
+- Unit: Milliseconds
+- This variable controls whether to enable the synchronously loading statistics feature. The default value `0` means that the feature is disabled. To enable the feature, you can set this variable to a timeout (in milliseconds) that SQL optimization can wait for at most to synchronously load complete column statistics. For details, see [Load statistics](/statistics.md#load-statistics).
 
 ### tidb_stmt_summary_history_size <span class="version-mark">New in v4.0</span>
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
 - Default value: `24`
 - Range: `[0, 255]`
 - This variable is used to set the history capacity of [statement summary tables](/statement-summary-tables.md).
 
 ### tidb_stmt_summary_internal_query <span class="version-mark">New in v4.0</span>
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
 - Default value: `OFF`
 - This variable is used to control whether to include the SQL information of TiDB in [statement summary tables](/statement-summary-tables.md).
 
 ### tidb_stmt_summary_max_sql_length <span class="version-mark">New in v4.0</span>
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
 - Default value: `4096`
 - Range: `[0, 2147483647]`
 - This variable is used to control the length of the SQL string in [statement summary tables](/statement-summary-tables.md).
 
 ### tidb_stmt_summary_max_stmt_count <span class="version-mark">New in v4.0</span>
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
 - Default value: `3000`
 - Range: `[1, 32767]`
 - This variable is used to set the maximum number of statements that [statement summary tables](/statement-summary-tables.md) store in memory.
 
 ### tidb_stmt_summary_refresh_interval <span class="version-mark">New in v4.0</span>
 
-- Scope: SESSION | GLOBAL
```

Comment:
This behavior change is intentional and only affects master. See: https://github.com/pingcap/tidb/blob/master/docs/design/2021-12-08-instance-scope.md